packages/next/src/server/dev/turbopack-utils.ts

```diff
@@ -64,13 +64,20 @@ export function isWellKnownError(issue: Issue): boolean {
 /// Print out an issue to the console which should not block
 /// the build by throwing out or blocking error overlay.
 export function printNonFatalIssue(issue: Issue) {
-  // Currently only warnings will be printed, excluding if the error source
-  // is coming from foreign (node_modules) codes.
+  const nonFatalissue = formatNonFatalIssue(issue)
+  if (nonFatalissue) {
+    Log.warn(formatIssue(issue))
+  }
+}
+
+export function formatNonFatalIssue(issue: Issue): string | null {
```

Comment:
This should not to filtering and formatting together.

Instead there should be `isRelevantWarning` and return boolean.